packages/grant-explorer/src/features/round/ViewRoundPage.tsx

```diff
@@ -129,43 +128,25 @@ function AfterRoundStart(props: {
   const [searchQuery, setSearchQuery] = useState("");
   const [projects, setProjects] = useState<Project[]>();
 
-  const [showChangeNetworkModal, setShowChangeNetworkModal] = useState(false);
-  const { switchNetwork } = useSwitchNetwork();
-  const { chain } = useNetwork();
+  const [showCartNotification, setShowCartNotification] = useState(false);
+  const [currentProjectAddedToCart, setCurrentProjectAddedToCart] =
+    useState<Project>({} as Project);
```

Comment:
Can a valid project actually be `{}`? If not, then we probably need `useState<null | Project>(null)`

Reply:
Indeed, however OOS for this ticket